File: Application/App/src/app/models/matchDetails/perk-style-dto.ts
import { PerkStyleSelectionDto } from "./perk-style-selection-dto";

export class PerkStyleDto {
    description : string = "";
    selections : PerkStyleSelectionDto[] = [];
    style : number = 0;


    public constructor(data : Partial<PerkStyleDto>){
        Object.assign(this,data);
    }
}

File: Backend/Backend/Models/Matches/PerkStyleDto.cs
namespace Backend.Models.Matches
{
    public class PerkStyleDto
    {
        public PerkStyleDto(string description, List<PerkStyleSelectionDto> selections, int style)
        {
            Description = description;
            Selections = selections;
            Style = style;
        }

        public string Description { get; set; }
        public List<PerkStyleSelectionDto> Selections { get; set; }
        public int Style { get; set; }
    }
}
